pkgs/tools/audio/mpdris2/default.nix

```diff
@@ -16,7 +16,7 @@ stdenv.mkDerivation rec {
   '';
 
   buildInputs = [ intltool autoreconfHook pythonPackages.wrapPython ];
-  propagatedBuildInputs = with pythonPackages; [ python pygtk dbus-python ];
+  propagatedBuildInputs = with pythonPackages; [ python pygtk dbus-python mutagen ];
```

Comment:
`propagatedBuildInputs` will propagate all elements of this list to other derivations, using this derivation in `buildInputs`